pr-data.csv

```diff
@@ -2663,7 +2663,7 @@ https://github.com/apache/incubator-kie-drools,8440dc2d4b95412a05a3c7b9501993e1a
 https://github.com/apache/incubator-kie-drools,8440dc2d4b95412a05a3c7b9501993e1ace9b496,drools-decisiontables,org.drools.decisiontable.parser.RuleWorksheetParseFromFileTest.testWorkbookParse,ID,Accepted,https://github.com/apache/incubator-kie-drools/pull/5610,
 https://github.com/apache/incubator-kie-drools,8440dc2d4b95412a05a3c7b9501993e1ace9b496,drools-impact-analysis/drools-impact-analysis-itests,org.drools.impact.analysis.example.ExampleUsageTest.testExampleUsage,ID,,,
 https://github.com/apache/incubator-kie-drools,8440dc2d4b95412a05a3c7b9501993e1ace9b496,drools-impact-analysis/drools-impact-analysis-itests,org.drools.impact.analysis.integrationtests.DrlSyntaxTest.testGlobal,ID,,,
-https://github.com/apache/incubator-kie-drools,8440dc2d4b95412a05a3c7b9501993e1ace9b496,drools-model/drools-model-codegen,org.drools.model.codegen.execmodel.AccumulateTest.testAccumulateMultipleFunctions,ID,,,
+https://github.com/apache/incubator-kie-drools,8440dc2d4b95412a05a3c7b9501993e1ace9b496,drools-model/drools-model-codegen,org.drools.model.codegen.execmodel.AccumulateTest.testAccumulateMultipleFunctions,ID,DeveloperWontFix,,https://github.com/apache/incubator-kie-drools/pull/6119
```

Comment:
Please separate this change from the other, as I can easily accept this one but not the other without more evidence.